temporalio/ui
# DT-826 - add i18n strings for event summary table

## Description & motivation 💭 
- add i18n strings for event summary table (this does not add i18n for event attributes, they are returned from the server and formatted to be displayed in the UI)
- also remove i18n from page.data - we don't need it, and it prevents us from using the `t` function in non-svelte components

### Screenshots (if applicable) 📸 
N/a
### Design Considerations 🎨 
N/a
## Testing 🧪 

### How was this tested 👻 


### Steps for others to test: 🚶🏽‍♂️🚶🏽‍♀️ 
Load the workflow details -> history page aka event summary, and ensure no missing strings
## Checklists
N/a
### Draft Checklist 

### Merge Checklist 

### Issue(s) closed 

## Docs
N/a
### Any docs updates needed?
